website/blog/2020-02-31-version-60.md

```diff
@@ -0,0 +1,61 @@
+
+> With this change, React Native apps will need to begin using AndroidX themselves. They cannot be used side-by-side in one app, so all of the app code and dependency code needs to be using one or the other.
+>
+> [matt-oakes](https://github.com/matt-oakes) on [discussions-and-proposals](https://github.com/react-native-community/discussions-and-proposals/issues/129)
+
+While your own native code will need to be migrated by you, [@mikehardy](https://github.com/mikehardy), [@cawfree](https://github.com/cawfree), and [@m4tt72](https://github.com/m4tt72) built a [clever tool](https://github.com/mikehardy/jetifier) to patch your `node_modules` while library maintainers are still working on releasing upgrades themselves. So if you find errors related to AndroidX migration, give this a shot.
+
+## Cocoapods by Default
+
+[CocoaPods are now part of React Native's iOS project](https://github.com/react-native-community/discussions-and-proposals/blob/master/proposals/0004-cocoapods-support-improvements.md). At the same time, the `podspec`s for the internal packages have changed to make them in sync with the Xcode projects, which will help with troubleshooting and debugging. Expect to make some straightforward changes to your `Podfile` as part of the upgrade to 0.60 to bring this exciting support.
+
+## Lean Core Removals
+
+**WebView** and **NetInfo** were previously extracted into separate repositories, and in 0.60 we’ve finished migrating them out of the React Native repository. Additionally, in response to community feedback about new App Store policy, **Geolocation** has been extracted. If you haven’t already, complete your migration by depending on [react-native-webview](https://github.com/react-native-community/react-native-webview), [@react-native-community/netinfo](https://github.com/react-native-community/react-native-netinfo), and [@react-native-community/geolocation](https://github.com/react-native-community/react-native-geolocation). Maintainers have made more than 100 commits to these repositories since extraction; we’re excited to see the community’s support!
+
+## Native Modules are now Autolinked
+
+The team working on [the cli](https://github.com/react-native-community/cli) has introduced major improvements to native module linking called [autolinking](https://github.com/react-native-community/cli/blob/master/docs/autolinking.md)! With it, the `react-native link` command is replaced by automatic native platform package linking, with no additional steps required. At the same time, the team overhauled the linking process in general. Let us know what you think using the hashtag [#autolinkingInRN](https://twitter.com/intent/tweet?hashtags=autolinkingInRN).
```

Comment:
the cli -> The React Native CLI

Reply:
Link is not replaced, it's still there but we believe it's not gonna be that useful now